PNG provide credential examples (#145)
bcgov/bc-vcpedia: credentials/bc-petroleum-and-natural-gas-title/vocabulary.md

# Vocabulary

> 

Reviewer: We should include a reference to the source of truth of these vocab entries, in this case, I believe it is the Act.

> 

Author: ```suggestion
_Some terms defined in this vocabulary are referencing terms defined in the [The Petroleum and Natural Gas Act (PNG Act)](https://www.bclaws.gov.bc.ca/civix/document/id/complete/statreg/00_96361_01#part1)._
```

Author: @krobinsonca Does this work for this purpose?

## Classes

### Director
An officer or employee of the ministry who is designated as the Director of Petroleum Lands by the minister

### LegalEntity
An entity registered in orgbook.

> ### TenureTitle

Reviewer 2: ```suggestion
### PetroleumNaturalGasTitle
```

> An officer or employee of the ministry who is designated as the Director of Petroleum Lands by the minister

Reviewer 2: ```suggestion
An officer or employee of the ministry who is designated as the Director of Petroleum Lands by the minister.
```

> An entity registered in orgbook.

Reviewer 2: ```suggestion
An entity registered in [OrgBook](https://orgbook.gov.bc.ca/search).
```

The majority of subsurface petroleum and natural gas (PNG) resources in British Columbia (B.C.) are owned by the Province. By entering into a tenure agreement with the Province, private industry can develop these resources. Tenure agreements are the mechanism used by the Province to give rights to petroleum and natural gas resources through issuance of Petroleum and Natural Gas Titles.

### TitleHolder

> The PNG Act describes a "holder of a location" as meaning "in accordance with the context, a permittee, licensee" . In this credential, the term title holder is used to reference a person in whose name a PNG title document is recorded in the division records. Title holders can be registered companies, and/or persons. A Title Holder will hold a percentage (%) interest in the title. Percent interest defines the percentage of interest allotted to each party named in the PNG title document. Percentage of ownership can be divided up to eight digits and should always equal 100% total. Lessee is defined by the PNG Act as "a person in whose name a lease is recorded in the division records".

Reviewer 2: ```suggestion
The PNG Act describes a "holder of a location" as meaning "in accordance with the context, a permittee, licensee". In this credential, the term Title Holder refers to a person whose name in a PNG title document is recorded in the division records. Title Holders can be registered companies, and/or persons. A Title Holder will hold a percent (%) interest in the title. Percent interest defines the percentage of interest allotted to each party named in the PNG title document. The percentage of ownership can be divided up to eight digits and should always equal 100% total. 
```

### Lease

> A subsisting lease issued under this Act

Reviewer 2: ```suggestion
A subsisting lease issued under the PNG Act.
```

### Leasee

> A person in whose name a lease is recorded in the division records

Reviewer 2: ```suggestion
Defined by the PNG Act as "a person in whose name a lease is recorded in the division records".
```

> A person in whose name a lease is recorded in the division records

Reviewer 2: ```suggestion
A person in whose name a lease is recorded in the division records.
```

### Licence

> A drilling licence or a licence under section 126 (3) or 130 (3)

Reviewer 2: ```suggestion
A drilling licence under [Part 5.1](https://www.bclaws.gov.bc.ca/civix/document/id/complete/statreg/96361_01#part5.1) of the PNG Act.
```

### Licensee

> A person in whose name a licence is recorded in the division records

Reviewer 2: ```suggestion
A person in whose name a licence is recorded in the division records.
```

### NaturalGas
All fluid hydrocarbons, before and after processing, that are not defined as petroleum, and any of the following that is produced from a well:

> carbon dioxide; helium; hydrogen; hydrogen sulphide;

Reviewer 2: ```suggestion
carbon dioxide; helium; hydrogen; hydrogen sulphide.
```

### Permit

> A subsisting exploration permit issued under this Act

Reviewer 2: ```suggestion
A subsisting exploration permit issued under the PNG Act.
```

### Permitee

> The person in whose name an exploration permit is recorded in the division records

Reviewer 2: ```suggestion
The person in whose name an exploration permit is recorded in the division records.
```

### Petroleum

> Crude petroleum and all other hydrocarbons, regardless of gravity, that are or can be recovered in liquid form from a pool through a well by ordinary production methods or that are or can be recovered from oil sand or oil shale

Reviewer 2: ```suggestion
Crude petroleum and all other hydrocarbons, regardless of gravity, that are or can be recovered in liquid form from a pool through a well by ordinary production methods or that are or can be recovered from oil sand or oil shale.
```

### PetroleumWell

> A well in which casing is run and that, in the opinion of the regulator, is producing or is capable of producing from a petroleum bearing zone

Reviewer 2: ```suggestion
A well in which casing is run and that, in the opinion of the regulator, is producing or is capable of producing from a petroleum-bearing zone.
```

### Well

> A hole in the ground
> (a) made or being made by drilling, boring or any other method to obtain petroleum or natural gas,
> (b) made or being made by drilling, boring or any other method to explore for, develop or use a storage reservoir for the storage or disposal of petroleum, natural gas, carbon dioxide, water produced in relation to the production of petroleum or natural gas, waste or any other prescribed substance,
> (c) used, drilled or being drilled to inject natural gas, water produced in relation to the production of petroleum or natural gas or other substances into an underground formation in connection with the production of petroleum or natural gas,
> (d) used to dispose of petroleum, natural gas, water produced in relation to the production of petroleum or natural gas, waste or any other prescribed substance into a storage reservoir, or
> (e) used, drilled or being drilled to obtain geological or geophysical information respecting petroleum or natural gas,

Reviewer 2: ```suggestion
A hole in the ground:
(a) made or being made by drilling, boring or any other method to obtain petroleum or natural gas;
(b) made or being made by drilling, boring or any other method to explore for, develop or use a storage reservoir for the storage or disposal of petroleum, natural gas, carbon dioxide, water produced in relation to the production of petroleum or natural gas, waste or any other prescribed substance;
(c) used, drilled or being drilled to inject natural gas, water produced in relation to the production of petroleum or natural gas or other substances into an underground formation in connection with the production of petroleum or natural gas;
(d) used to dispose of petroleum, natural gas, water produced in relation to the production of petroleum or natural gas, waste or any other prescribed substance into a storage reservoir; or
(e) used, drilled or being drilled to obtain geological or geophysical information respecting petroleum or natural gas.
```

### Zone

> A stratum or strata designated by the regulator as a zone generally or for a designated area or a specific well, or a stratum or strata designated by the director for the purposes of the administration of petroleum and natural gas rights or storage and disposal rights.

Reviewer 2: ```suggestion
A stratum or strata designated by the regulator as a zone generally for a designated area or a specific well, or a stratum or strata designated by the director for the purposes of the administration of petroleum and natural gas rights or storage and disposal rights.
```

### Tract

> A Tract is an area of land within a title defined by locations sharing identical rights. This attribute has all of the tract(s) information contained within a title. See Tracts Appendix below for a detailed description of the information included.

Reviewer 2: ```suggestion
A Tract is an area of land within a title defined by locations sharing identical rights. This attribute has all of the tract(s) information contained within a title. 
```

## Properties

### interest

> an undivided interest in a location

Reviewer 2: ```suggestion
an undivided interest in a location.
```

### titleType
This field specifies the type of title based on the PNG Act. Title types can include the following: 1. Lease (Part 6); 2. Permit (Part 5); 3. Drilling Licence (Part 5.1), and 4. Storage Reservoir Licence (Part 14.130). Each title has only one title type.

### titleNumber

> A number is used to uniquely identify a title, assigned based on administrative policy by the TRSB. The title number is associated with the Exploration Licence outlined in Section 126, part 14.126 of the PNG Act.

Reviewer 2: ```suggestion
A number is used to uniquely identify a title, assigned based on administrative policy by the Tenure and Resource Stewardship Branch (TRSB). The title number is associated with the Exploration Licence outlined in Section 126, Part 14.126 of the PNG Act.
```

### originType
This field specifies the type of origin based on the PNG Act. Origin types can include the following: 1. Lease (Part 6); 2. Permit (Part 5); 3. Drilling Licence (Part 5.1), and 4. Storage Reservoir Licence (Part 14.130). Each title has only one origin type.

### originNumber
A number is used to uniquely identify an origin, assigned based on administrative policy by the TRSB.

### area

> This attribute is the surface area in planned use, measured in hectares. This includes 3 dimensional layers.

Reviewer 2: ```suggestion
This attribute is the surface area in planned use, measured in hectares. This includes three dimensional layers.
```

### term
This describes the length of time (term) granted for each title, expressed in YYYY. This date does not change.

### caveats

> Caveats provide information and guidance to the tenure holder that will assist in activity planning by identifying potential access restrictions. Caveats will also flag concerns identified through pre-tenure consultation and may recommend engagement with First Nations, stakeholders, and other agencies as appropriate. Caveats often point to relevant statute and policy and are not binding or enforced by the Ministry.

Reviewer 2: ```suggestion
Caveats provide information and guidance to the title holder that will assist in activity planning by identifying potential access restrictions. Caveats will also flag concerns identified through pre-tenure consultation and may recommend engagement with First Nations, stakeholders, and other agencies as appropriate. Caveats often point to relevant statutes and policy and are not binding or enforced by the ministry.
```
